.travis.yml

```diff
@@ -5,51 +5,52 @@ dist: xenial
 
 matrix:
   include:
-    - os: linux
-      jdk: openjdk11
-  allow_failures:  
-      - os: osx
-        osx_image: xcode9
- 
-script: ./gradlew clean build installToRoot test compileSlowtest datatest pfinttest allReports buildDist buildNsis pcgenRelease
+  - os: linux
+    jdk: openjdk11
+  allow_failures:
+  - os: osx
+    osx_image: xcode9
+
+script: "./gradlew clean build installToRoot test compileSlowtest datatest pfinttest
+  allReports buildDist buildNsis pcgenRelease"
 # Do not use itest, requires release notes
 
 before_install:
-    - sudo apt-get update
-    - sudo apt-get install lib32z1 lib32ncurses5 nsis
-    - export PERL_MM_USE_DEFAULT=1
-    - chmod +x gradlew
+- sudo apt-get update
+- sudo apt-get install lib32z1 lib32ncurses5 nsis
+- export PERL_MM_USE_DEFAULT=1
+- chmod +x gradlew
     - if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sudo apt-get install scons zlib1g-dev libcppunit-dev; fi
-    - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew update; fi
-    - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew install https://sourceforge.net/p/nsis/code/HEAD/tree/web-scripts/trunk/cppunit.rb?format=raw; fi
-    - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew install scons mingw-w64 zlib; fi
- 
+- if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew update; fi
+- if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew install https://sourceforge.net/p/nsis/code/HEAD/tree/web-scripts/trunk/cppunit.rb?format=raw;
+  fi
+- if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew install scons mingw-w64 zlib; fi
 install:
-    - wget -O /tmp/zlib.zip "https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib128-dll.zip?r=&ts=1403402496&use_mirror=hivelocity"
-    - mkdir /tmp/zlib
-    - unzip /tmp/zlib.zip -d /tmp/zlib
- 
+- wget -O /tmp/zlib.zip "https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib128-dll.zip?r=&ts=1403402496&use_mirror=hivelocity"
+- mkdir /tmp/zlib
+- unzip /tmp/zlib.zip -d /tmp/zlib
+
 before_cache:
-    - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-    - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
   directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-    
+  - "$HOME/.gradle/caches/"
+  - "$HOME/.gradle/wrapper/"
+
 before_deploy:
-    # Build release files
-    - ./gradlew pcgenRelease
+- "./gradlew pcgenRelease"
 
 deploy:
-    provider: releases
-    #Replace this with a real token
-    api_key: "GITHUB OAUTH TOKEN"
+  provider: releases
+  api_key:
+    secure: vcWcTepSQfgZTbLrC2jbJKLFB8pho72dz00vqUyWjtilvLcbOYVgVWgE5U5zjPV+VS5j4cj93sMWv74dpwamXGv/kIld7fQlpVkMELtK5AkRa2bjOMi0BHk54K8gcGOG//Gen/3gdBJfV1kYcsOhC2Rc4t4pcY2rkW534NLUxdg=
```

Comment:
Are we leaking a secret here?

Reply:
It's how the travis set this up. That was a script add.